### PR TITLE
Fix events cards title overflow

### LINKED ISSUE #1124: Events Cards overflow
### Steps to reproduce the problem
Go to the Qiskit events page and have a card with a long title.

### What is the current behavior?
Title overflows the card.
2020-11-17 16_30_48-Qiskit Events

### What is the expected behavior?
The title must fit into the card.

### PR DESCRIPTION
**Pull request by korgan00**
Fix #1124

### CODE REVIEW

File: components/ui/AppCard.vue [hunk 1]
```diff
@@ -93,27 +95,21 @@ export default class extends Vue {
   }
 
   &__title {
-    flex: 0 0 auto;
+    flex: 1 1 auto;
```
Comment (delapuente) [inline components/ui/AppCard.vue:98]: Allow the title to grow with `flex: 1`...

File: components/ui/AppCard.vue [hunk 2]
```diff
@@ -93,27 +95,21 @@ export default class extends Vue {
...
     @include type-style('productive-heading-02');
   }
 
-   &__tags {
-    width: 20rem;
+  &__tags {
     display: flex;
     flex-wrap: wrap;
+    flex: 1 1 auto;
```
Comment (delapuente) [inline components/ui/AppCard.vue:105]: ...and prevent the tags from growing with `flex: 0`.

File: pages/events/index.vue [hunk 3]
```diff
@@ -332,6 +334,14 @@ export default class extends QiskitPage {
     }
   }
 
+  &__empty-card-description {
```
Comment (delapuente) [inline pages/events/index.vue:337]: How is this change related to this fix?

**Review by delapuente — CHANGES_REQUESTED** [review]
Nice catch here. Please, review that you're addressing the bug described in the issue and only that. Move other changes to a different PR.

Also take a look at my comments since the problem you're describing in the issue is easily solvable with a couple of changes in the `flex` behaviour.

File: pages/events/index.vue (hunk 3 above)
Comment (korgan00) [reply pages/events/index.vue:337]: I took commits directly from the advocates branch. This issue was fixed in that branch as a side effect of fixing other issues.
That fixes changed the empty card too, and it was necessary to fix it. I do it this way to be faster and make easier the future merge with that branch.

**Comment by korgan00** [comment]
I deleted all the unrelated code to fix the overflow and respect the design.
I forced push to have only one commit to prevent multiple conflicts.

**Review by delapuente — APPROVED** [review]
Nice done.

**Pull request merged**